cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -257,6 +257,17 @@ void Run(const std::vector<string>& bag_filenames) {
     }
 
     bag.close();
+
+    if (FLAGS_keep_running) {
```

Comment:
I'll be on vacation most of next week, but I think the week after we can do a hangout.